coriolisclient/tests/cli/test_utils.py

```diff
@@ -0,0 +1,311 @@
+        )
+
+        obj = mock.Mock()
+        obj.prop_name.to_dict.return_value = \
+            {"key1": "value1", "key2": "value2"}
+
+        result = utils.format_json_for_object_property(obj, "prop_name")
+
+        self.assertEqual(
+            '{\n  "key1": "value1",\n  "key2": "value2"\n}',
+            result
+        )
+
+        obj.prop_name = None
+
+        result = utils.format_json_for_object_property(obj, "prop_name")
+
+        self.assertEqual(
+            '{}',
+            result
+        )
+
+    def test_validate_uuid_string(self):
+        result = utils.validate_uuid_string(
+            "12345678-9ABC-DEF1-2345-6789abcdef12")
+
+        self.assertEqual(
+            True,
+            result
+        )
+
+        result = utils.validate_uuid_string(
+            "123456789ABCDEF")
+
+        self.assertEqual(
+            False,
+            result
+        )
+
+    def test_add_args_for_json_option_to_parser(self):
+        parser = mock.Mock()
+
+        result = utils.add_args_for_json_option_to_parser(
+            parser, "option_name")
+
+        self.assertEqual(
+            parser,
+            result
+        )
+
+    def test_get_option_value_from_args(self):
+        args = mock.MagicMock()
+        args.option_name = '{"option": "raw_value"}'
+        args.option_name_file.__enter__.return_value.read.return_value = \
+            '{"option": "file_value"}'
+
+        result = utils.get_option_value_from_args(args, "option-name")
+
+        self.assertEqual(
+            {'option': 'raw_value'},
+            result
+        )
+
+        args.option_name = None
+
+        result = utils.get_option_value_from_args(args, "option-name")
+
+        self.assertEqual(
+            {'option': 'file_value'},
+            result
+        )
+
+    def test_get_option_value_from_args_no_value(self):
+        args = mock.Mock()
+        args.option_name = None
+        args.option_name_file = None
+
+        result = utils.get_option_value_from_args(
+            args, "option-name", error_on_no_value=False)
+
+        self.assertEqual(
+            None,
+            result
+        )
+
+        self.assertRaises(
+            ValueError,
+            utils.get_option_value_from_args,
+            args,
+            "option-name"
+        )
+
+    def test_get_option_value_from_args_json_value_error(self):
+        args = mock.Mock()
+        args.option_name = "invalid"
+        args.option_name_file = None
+
+        self.assertRaises(
+            ValueError,
+            utils.get_option_value_from_args,
+            args,
+            "option-name"
+        )
+
+    @ddt.data(
+        {
+            "global_scripts": None,
+            "instance_scripts": None,
+            "expected_result":
+            {
+                "global": {},
+                "instances": {}
+            }
+        },
+        {
+            "global_scripts": ["linux="],
+            "instance_scripts": ["linux="],
```

Comment:
Please don't input os_type as instance. It should be something like `instance_1`